service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/testingsupport/IntegrationTestConstants.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.fpl.testingsupport;
+
+public interface IntegrationTestConstants {
```

Comment:
there is already uk.gov.hmcts.reform.fpl.Constants for things like this

Reply:
I wanted to make this accessible only in Integration test.

Reply:
as this is only applicable in that context's configuration